docs/guide/introduction.rst

```diff
@@ -0,0 +1,296 @@
+- A ``.gitignore`` file so you don't accidentally push the ``config.py`` file to a version control system like GitHub.
+
+There is also ``newcog`` which does the following:-
+
+.. code-block:: shell
+    :emphasize-lines: 1
+
+    $ python -m discord newcog CoolCog CoolBot/cogs
+    > successfully made cog at CoolBot/cogs/CoolCog.py
+
+This tool creates a new file within the passed directory with the provided name, and again this is a blanket implementation of an :ref:`extension <ext_commands_extensions>` file for use with the :meth:`~ext.commands.Bot.load_extension` method.
+
+The file imports the necessary items and defines a Cog class that you can extend, and adds the necessary ``setup`` method for the extension to function correctly.
+
+
+Running Your New Bot
+---------------------
+
+Now that you have your code ready, go to your terminal and ``cd`` into your project directory:
+
+.. code-block:: shell
+
+    $ cd my_bot_folder
+
+Activate the virtual environment (if you made one):
+
+.. code-block:: shell
+
+    $ source .venv/bin/activate  # for Linux users
+    $ .\.venv\Scripts\activate   # for Windows users
+
+And run your bot!
+
+.. code-block:: shell
+
+    (.venv) $ python your_bot.py
+    Ready! I am Documentation#7968 and my ID is 699701272739053589
+
+.. _securing_token:
+
+Securing your bot's token
+--------------------------
+
+Your bot's token is **very important**. If a malicious user obtains it, they can delete your private guilds or get you and your bot banned.
+This short sub-guide shows a few basic ways to secure your token, so it's not publicly obtainable.
+
+.. note::
+
+    This will not go over securing your bot's host system (i.e. VPS). Make sure your system is private and only you and authorized users can access it.
+
+1. Create a config file.
+
+    There are a few options for this, and you can choose whichever you prefer:
+
+    - JSON, YAML, TOML
+    - Python (you can import the config!)
+    - Environment Variable (either on your system, or with a ``.env`` file)
+
+    For this example, we will use a Python file.
+
+2. Store your token and other secrets here.
+
+    .. code-block:: python
+
+        token = "123"
+        database_password = "hello_world"
+
+3. Import your config module into your main bot file
+
+    .. code-block:: python
+
+        import config
+        bot = commands.Bot(command_prefix="!", intents=discord.Intents.default())
+        bot.run(config.my_token)
+
+4. If you store your bot's code on a public repository (e.g. GitHub), you can create a ``.gitignore`` file to prevent pushing the config file:
+
+    .. code-block:: python
+
+        # file: .gitignore
+        config.py
+
+    .. code-block:: shell
+
+        $ git add .gitignore
+        $ git commit -m "Add .gitignore"
+        $ git status
+        # notice how there is no more ``config.py``!
+        On branch master
+        no changes to commit, working tree clean
+
+    .. note::
```

Comment:
Honestly the note directly doesn't seem to make sense here to me. It could just be an regular sentence but i'm not so sure